contracts/Super1155.sol

```diff
@@ -320,7 +320,7 @@ contract Super1155 is PermitControl, ERC165, IERC1155, IERC1155MetadataURI {
     } else if (hasRightUntil(_msgSender(), bytes32(_id), _right)
       > block.timestamp) {
       _;
-    }
+    } 
```

Comment:
Added a single space? Let's cut this change out of the PR to keep the commit history as simple as possible.

Reply:
Alright. This was to indicate the place for the revert statement. But I'll remove this